src/main/java/lotto/domain/Rank.java

```diff
@@ -2,24 +2,60 @@
 
 public enum Rank {
 
+	MISS(0, 0),
 	FIFTH(3, 5000),
 	FOURTH(4, 50000),
 	THIRD(5, 1500000),
+	SECOND(5, 30000000),
 	FIRST(6, 2000000000);
 
-	private final int matchedNumber;
+	private final int countOfMatch;
 	private final int winningAmount;
```

Comment:
금액을 의미하는 winningAmount 도 Money 클래스로 만들어볼 수 있을 것 같아요 :)

```suggestion
	private final Money winningAmount;
```